### PR TITLE
Weather App with API

### PR DESCRIPTION
**Pull request by eliasecasta**
In this project I created a simple weather app that gets data from the Open Weather API in order to display the user the current temperature of the city that is being queried.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello  👋

**You've done pretty well here fixing a few issues raised in the previous review. 👌**

However, let's make a couple of changes.

- Please refer to comments on the code.

_OPTIONAL_
- [ ] Consider initially accessing a user's location and getting the weather for that location to show a default weather state.

- [ ] You can also consider showing a loading indicator while retrieving the weather info.


### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<!-- <!DOCTYPE html> -->
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <title>Weather App</title>
+    <link href="https://cdn.jsdelivr.net/gh/gitbrent/bootstrap-switch-button@1.1.0/css/bootstrap-switch-button.min.css"
```
Comment (mikenath223) [inline src/index.html:9]: - [ ] Kindly delete this index.html file under the src directory since it serves no purpose.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+  if (celOrFahr === 'celsius') {
+    cityTemperatureType.append('C');
+  } else {
+    cityTemperatureType.append('F');
+  }
+
+  cityTime.append(currentTimeReadable);
+
+  return currentTime._i;
```
Comment (mikenath223) [inline src/index.js:80]: - [x]  As this is what is causing the eslint error then consider disabling the eslint no-underscore-dangle rule for this particular line. Only do this if there is no other way to get the project to work without this underscore.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+});
+
+async function initApp() {
+  const city = await getCity();
+  const unixTime = await displayDetails(city);
+  const dayNight = await dayOrNight(city, unixTime);
+
+  if (dayNight === 'night') {
+    $('.main-title').css('color', 'white');
+    $('.row h1').css('color', 'white');
```
Comment (mikenath223) [inline src/index.js]: - [x] Please change this to es6 arrow function syntax. Make this change on other regular functions in the project too.

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+  if (celOrFahr === 'celsius') {
+    API = `https://api.openweathermap.org/data/2.5/weather?q=${cityName}&units=metric&appid=934ea5f8bc3f198fabf59b607a2fcc71`;
+  } else {
+    API = `https://api.openweathermap.org/data/2.5/weather?q=${cityName}&units=imperial&appid=934ea5f8bc3f198fabf59b607a2fcc71`;
+  }
+
+  const response = await fetch(API);
+  const city = await response.json();
+
+  return city;
```
Comment (mikenath223) [inline src/index.js]: - [x] Kindly handle async fetch errors here either with a try/catch block or chaining a `.catch` to fetch requests.

**Comment by mikenath223** [comment]
- [x] Kindly switch the default branch to the `develop` branch. It's important you follow the GitHub workflow guides shared by other students.

File: src/index.html (hunk 1 above)
Comment (eliasecasta) [reply src/index.html:9]: It gets compiled by webpack to produce the html file located at dist.

File: src/index.js (hunk 3 above)
Comment (eliasecasta) [reply src/index.js]: Can you please explain me why should I change them to arrow functions? All I found on the internet is that arrow functions are useful for one liner functions, that is the only relevant use that I could find about them.

Comment (eliasecasta) [reply src/index.js]: Other than that they just look ugly to me and they even add clutter to the function statement.

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @e71az :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**